modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/TableRowSkinTest.java

```diff
@@ -244,6 +247,47 @@ public void tableRowWithFixedSizeShouldIgnoreVerticalPadding() {
         assertEquals(height, cell.getHeight(), 0);
     }
 
+    /**
+     * When we make an invisible column visible we expect the underlying cells to be visible, e.g. as width > 0.
```

Comment:
This only applies to the case of fixed cell size set (this test will fail if you comment out line 256), so maybe you could clarify the comment?

Reply:
Yes! Will do.